src/test/java/gameframework/motion/MoveStrategyRandomTest.java

```diff
@@ -49,5 +51,18 @@ public void goDown() throws Exception {
 		setRandom(3);
 		assertDown();
 	}
+	
+	@Test
+	public void goodSpeed() throws Exception {
+
+		MoveStrategyRandom strat = new MoveStrategyRandom(4);
+		
+		assertEquals(strat.getSpeed(), 4);
```

Comment:
`assertEquals` first argument is the `expected` one and the 2nd is the `actual` one. Please swap them all.